### PR TITLE
Merge master dev 2021.05.26

### PR DESCRIPTION
**Pull request by iulianpascalau**

### CODE REVIEW

**Review by sasurobert — APPROVED** [review]

File: cmd/storer2elastic/main.go [hunk 1]
```diff
@@ -308,7 +308,7 @@ func startStorer2Elastic(ctx *cli.Context) error {
 			RatingsProcessor:          ratingsProcessor,
 			RatingConfig:              ratingsConfig,
 			StartingEpoch:             uint32(flagsValues.startingEpoch),
-			WaitingListFixEnableEpoch: nodeConfig.GeneralSettings.WaitingListFixEnableEpoch,
+			WaitingListFixEnableEpoch: 0, //TODO(next PR): remove the storer2elastic app
```
Comment (miiu96) [inline cmd/storer2elastic/main.go:311]: add a JIRA task for this

**Review by miiu96 — APPROVED** [review]

File: cmd/storer2elastic/main.go (hunk 1 above)
Comment (iulianpascalau) [reply cmd/storer2elastic/main.go:311]: will be done automatically, without Jira task :)

Comment (iulianpascalau) [reply cmd/storer2elastic/main.go:311]: https://github.com/ElrondNetwork/elrond-go/pull/3121

**Review by gabi-vuls — APPROVED** [review]
System test passed

**Pull request merged**